libvast/include/vast/http_api.hpp

```diff
@@ -142,34 +144,95 @@ struct rest_response {
   caf::error detail_ = {};
 };
 
-/// Go through the provided parameters; discard those that are not understood by
-/// the endpoint and attempt to parse the rest to the expected type.
-auto parse_endpoint_parameters(
-  const vast::rest_endpoint& endpoint,
-  const detail::stable_map<std::string, std::string>& params)
-  -> caf::expected<vast::record>;
-
 /// Used for serializing an incoming request to be able to send it as a caf
 /// message.
 class http_request_description {
 public:
   /// Unique identification of the request endpoint.
   std::string canonical_path;
 
-  /// Request parameters. (unvalidated)
-  /// Query parameters, path parameters, and JSON body parameters
-  /// all get merged into this map.
+  /// Query parameters, path parameters, x-www-urlencoded body parameters
   detail::stable_map<std::string, std::string> params;
```

Comment:
Let's comment that out as long as we aren't using it. It is currently untested and offering it gives a false assurance.